release.nix

```diff
@@ -59,13 +65,16 @@ let
       PKG_IGNORE_TAG = 1;
       buildPhase = ''
         cp ${./package.json} package.json
+        mkdir -p output/prebuilds
+        cp -r node_modules/leveldown/prebuilds/darwin-${arch} output/prebuilds/darwin-${arch}
         pkg . \
           --targets macos-${arch} \
           --no-bytecode \
-          --output out
+          --public-packages "*" \
+          --output output/TypeScript-Demo-Lib
```

Comment:
Why are we creating a `output` directory?

The `$out` is obviously for the final executable. But I see you're putting `prebuilds` into the `output`. Is that not possible with just the `./` directory?

Reply:
- changing `output` to `pkg`. relevant to above comment.